src/pages/index.tsx

```diff
@@ -34,6 +34,7 @@ export const Home = () => {
       >
         <Box
           component={SumoLogicDocsLogo}
+          alt="Sumo Logic Docs logo"
```

Comment:
<img width="649" alt="alt" src="https://user-images.githubusercontent.com/56411016/219798790-21712047-9275-4261-b26d-c370f5bddb43.png">

Reply:
I believe this SVG should have `role='img'` with an `alt` attribute of only `"Sumo Logic Docs"`.  That said, I think this one should just be `aria-hidden=true` since this is more decorative, but the logo in the header bar should have the appropriate attributes.

Reply:
Thank you!